frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx

```diff
@@ -200,6 +203,19 @@ const GraphicalSelector = () => {
     }
   };
 
+  const zoomIn = () => {
+    if (!graph) return;
+    const zoom = graph.getZoom();
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    zoom >= 1 ? graph.zoomTo(zoom + 0.5) : graph.zoomTo(zoom + 0.1);
```

Comment:
what is the `>= 1` check for?

Reply:
so basically when the zoom becomes < 1 it zooms really far out in proportion to if the zoom is 2, 3, 4 etc. Hence at that point we need to reduce the zoom amount by 0.1. Its based on the builtin zoom feature